### PR TITLE
added toggle for alignment & color(hex)

### PR DESCRIPTION
**Pull request by d4n3x**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,7 +92,17 @@
                         "B"
                     ],
                     "default": "GB"
-                }
+                },
+				"resmon.alignLeft": {
```
Comment (Njanderson) [inline package.json]: Is it just on my screen that there're alignment issues with the JSON itself?

I think I'll just bring it in then I'll fix the alignment on my next iteration, unless you want to.

I also want to change this to an enum from a boolean to accommodate future values (maybe centered alignment?) or maybe we will have more locations that this information can be displayed.

File: src/extension.ts [hunk 2]
```diff
@@ -253,6 +261,9 @@ class ResMon {
             // Get the display of the requested resources
             let pendingUpdates: Promise<string | null>[] = resources.map(resource => resource.getResourceDisplay());
 
+            //change color
```
Comment (Njanderson) [inline src/extension.ts]: Nit comment style:
```
// Change color
```

File: src/extension.ts [hunk 3]
```diff
@@ -214,12 +214,20 @@ class DiskSpace extends Resource {
...
     private _delimiter: string;
     private _updating: boolean;
 
     constructor() {
+        if(this._config.get('alignLeft'))
+            this._statusBarItem = window.createStatusBarItem(StatusBarAlignment.Left);
+        else
+            this._statusBarItem = window.createStatusBarItem(StatusBarAlignment.Right);
+        
+        //Get color
```
Comment (Njanderson) [inline src/extension.ts]: Nit comment style:
```
// Get color
```

File: src/extension.ts [hunk 4]
```diff
@@ -214,12 +214,20 @@ class DiskSpace extends Resource {
...
 
 class ResMon {
-    private _statusBarItem: StatusBarItem = window.createStatusBarItem(StatusBarAlignment.Left);
+    private _statusBarItem: StatusBarItem;
     private _config: WorkspaceConfiguration;
     private _delimiter: string;
     private _updating: boolean;
 
     constructor() {
+        if(this._config.get('alignLeft'))
```
Comment (Njanderson) [inline src/extension.ts]: nit spacing and mandatory braces

```
if (this._config.get('alignLeft'))  {
...
}
```

**Comment by d4n3x** [comment]
I am totally fine with the changes but i am a bit in hurry atm. Would be nice if u could change that! :-)

File: package.json (hunk 1 above)
Comment (d4n3x) [reply package.json]: I'm totally fine with that.

**Comment by eduncan911** [comment]
@d4n3x please update!  This would sweet to have merged.  :)

File: src/extension.ts (hunk 2 above)
Comment (letmaik) [reply src/extension.ts]: I would probably just remove this comment...
Does the default color work as expected for both the light and dark theme?

File: src/extension.ts (hunk 3 above)
Comment (letmaik) [reply src/extension.ts]: Remove comment instead.

File: src/extension.ts (hunk 4 above)
Comment (letmaik) [reply src/extension.ts]: Maybe to make this easier for contributors there could be a style file for automatic formatting?

File: package.json [hunk 5]
```diff
@@ -92,7 +92,17 @@
...
+					"type": "boolean",
+					"default": true,
+					"description": "Toggles the alignment of the status bar."
+				},
+				"resmon.color": {
+					"type": "string",
+					"default": "#FFFFFF",
+					"description": "Color of the status bar."
```
Comment (letmaik) [inline package.json]: I'm assuming this is text color, maybe worth mentioning in the description.

**Pull request merged**